umbral/utils.py

```diff
@@ -42,3 +43,16 @@ def kdf(ecpoint, key_length):
 
 def get_curve_keysize_bytes(curve):
     return (curve.key_size + 7) // 8
+
+def get_field_order_size_in_bytes(curve):
+
```

Comment:
Only thing I would change is to drop the 'in' so it's `get_field_order_size_bytes`, but this looks great!